demos/uiOptions/index.html

```diff
@@ -85,6 +85,34 @@
                     "tocTemplate": "../../src/components/tableOfContents/html/TableOfContents.html",
                     "ignoreForToC": {
                         "overviewPanel": ".flc-overviewPanel"
+                    },
+                    selectors: {
+                        languageLinkEnglish: ".flc-languages-en",
```

Comment:
It's not uncommon for integrators to use this demo as a starting point for implementing UIO on their own site. We may want to split the localization stuff into a separate demo or example. Additionally all the parts that are specific to the language buttons should probably be factored off into their own grade. This should help distinguish between the localizable UIO and an implementation of something to change the language.